reports/jeschkies/unit/3/exception

```diff
@@ -17,7 +17,7 @@ Traceback (most recent call last):
     return (yield from handler(request))
   File &quot;/Users/karsten/.local/share/virtualenvs/unit-fFY_xGcI/lib/python3.6/site-packages/aiohttp_jinja2/__init__.py&quot;, line 88, in wrapped
     context = yield from coro(*args)
-  File &quot;/Users/karsten/Documents/unit/unitfm/main.py&quot;, line 41, in view_junit
+  File &quot;/Users/karsten/Documents/unit/fm.unit.unitfm/main.py&quot;, line 41, in view_junit
```

Comment:
What is this file for and is it checked in?

Reply:
`./reports` will go. It's just for some tests. I think IntelliJ replaced everything text of `unitfm` it could find.

Reply:
No, I mean, what is this `exception` file and why is it commited?